utils/dockerfile_generator.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
```

Comment:
@nadouani Hey it might help if you start from here... This is the script that I used to build those dockerfiles.

It uses the relevant `requirements.txt` to create a list of Alpine dependencies, then updates the file if any new changes are detected.

Reply:
@jeromeleonard Psst over here ☝️